pkg/kv/kvserver/replica_proposal_quota.go

```diff
@@ -164,9 +164,14 @@ func (r *Replica) updateProposalQuotaRaftMuLocked(
 		) {
 			return
 		}
-
-		// Only consider followers that that have "healthy" RPC connections.
-		if err := r.store.cfg.NodeDialer.ConnHealth(rep.NodeID, r.connectionClass.get()); err != nil {
+		// At this point, we know that either we communicated with this replica
+		// recently, or we became the leader recently. The latter case is ambiguous
+		// w.r.t. the actual state of that replica, but it is temporary.
+		//
+		// We do a second check using the NodeLiveness to increase the chance
+		// of disambiguating here, and also to catch other slowness that may be
+		// surfaced in the liveness but not in isFollowerActiveSince.
```

Comment:
I think we can reword this to allude to the fact that we don't want a node that isn't heartbeating (for whatever reason) to hold up the range.

Reply:
Not needed, as we decided on not adding the `NodeLiveness` check.